### ISSUE TITLE
What does m mean? and locc[:, :, :, 0]

### ISSUE THREAD

**Issue by qiannvMao** [shen1994/DeepFaceRestruction#2]
locc = tf.square(y - pos)
        loss_x = tf.reduce_mean(tf.multiply(locc[:, :, :, 0], m))
        loss_y = tf.reduce_mean(tf.multiply(locc[:, :, :, 1], m))
        loss_z = tf.reduce_mean(tf.multiply(locc[:, :, :, 2], m))
        loss = loss_x + loss_y + loss_z

**Comment by JianBin-Liu** [comment]
m means the weight mask,locc[:,:,:,0], first place means the number of train data, so fourth place means the image's channel
